partner_defaulter/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2016 ClearCorp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Partner defaulter',
+    'summary': 'Identify slow payers',
```

Comment:
Fixed
El mié., 29 jun. 2016 a las 15:01, Glen Sojo (<notifications@github.com>)
escribió:

> In partner_defaulter/__openerp__.py
> <https://github.com/ClearCorp/odoo-clearcorp/pull/649#discussion_r69026852>
> :
>
> > @@ -0,0 +1,26 @@
> > +# -*- coding: utf-8 -*-
> > +# © 2016 ClearCorp
> > +# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
> > +
> > +{
> > +    'name': 'Partner defaulter',
> > +    'summary': 'Identify slow payers',
>
> Change summary to *defaulters* instead of *slow payers*
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ClearCorp/odoo-clearcorp/pull/649/files/03308577ba1b2712f51db49dc5f2166f47076032#r69026852>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APifad16S0GB4UPnyN0eWeOwCm6yd436ks5qQtzxgaJpZM4JBdFc>
> .
>
-- 

Lesmed Gutiérrez · ClearCorp
Odoo Developer

lesmed.gutierrez@clearcorp.cr <glen.sojo@clearcorp.cr>
CR: +(506) 4000 2677
US: +1 (786) 472-4267
http://clearcorp.cr